src/tools/annotation/CircleRoiTool.js

```diff
@@ -384,8 +386,8 @@ function _createTextBoxContent(
     const hasStandardUptakeValues = meanStdDevSUV && meanStdDevSUV.mean !== 0;
     const unit = _getUnit(modality, options.showHounsfieldUnits);
 
-    let meanString = `avg: ${toGermanNumberStringTemp(mean)} ${unit}`; //`Mean: ${numbersWithCommas(mean.toFixed(2))} ${unit}`;
-    const stdDevString = `sd: ${toGermanNumberStringTemp(stdDev)} ${unit}`; //`Std Dev: ${numbersWithCommas(stdDev.toFixed(2))} ${unit}`;
+    let meanString = `avg: ${toGermanNumberStringTemp(mean)} ${unit}`; // `Mean: ${numbersWithCommas(mean.toFixed(2))} ${unit}`;
+    const stdDevString = `sd: ${toGermanNumberStringTemp(stdDev)} ${unit}`; // `Std Dev: ${numbersWithCommas(stdDev.toFixed(2))} ${unit}`;
```

Comment:
Why do we have these comments?

Reply:
Assuming the German localisation has been a temporary solution, therefore the earlier codes were remained as commented by Kathi

Reply:
I would then opt to let Git take care of persisting the code history and not keeping it as comments 😄